src/frontend/apps/desk/src/features/teams/components/PanelTeams.tsx

```diff
@@ -0,0 +1,129 @@
+import { Loader } from '@openfun/cunningham-react';
+import React, { Fragment, useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Text } from '@/components';
+import { InfiniteScroll } from '@/components/InfiniteScroll';
+
+import { TeamResponse, useTeams } from '../api/useTeams';
+import { useTeamStore } from '../store/useTeamsStore';
+
+import { PanelTeam } from './PanelTeam';
+
+type DictTeams = {
+  [page: string]: TeamResponse[];
+};
+
+export const PanelTeams = () => {
+  const { t } = useTranslation();
+
+  const [dictTeams, setDictTeams] = useState<DictTeams>({});
+  const [page, setPage] = useState(1);
+  const ordering = useTeamStore((state) => state.ordering);
```

Comment:
smth as `const {ordering} = useTemaStore()`would not work? Sorry I am not a zustand expert

Reply:
https://github.com/numerique-gouv/people/pull/31/commits/669aa39320299052bfcb86f2201f2b2bc2c32671